deid/dicom/parser.py

```diff
@@ -81,6 +91,8 @@ def __init__(
         # Deid can be a recipe or filename
         if not isinstance(recipe, DeidRecipe):
             recipe = DeidRecipe(recipe)
+
+        self.from_file = from_file
```

Comment:
Is there any reason this from_file should be saved with the class? E.g., isn't it just used for the load, and if so, would it make sense to pass there?

Also, is there a better way to determine this? E.g., if it's not a file that exists could we not just check for the type of the variable and not require the extra param?

Reply:
The reason to pass it in the constructor is that __init__ runs self.load.

You are right that a more intelligent way of doing it is checking if the variable exists and if it does not, just skip the necessary lines. I will modify the PR :smile: